elasticsearch/data/.keep

```diff
@@ -0,0 +1 @@
+just to not accidentally delete this directory, it's required by the /index-vectors.sh script
```

Comment:
Why does this folder need to be here? Can't `index-vectors.sh` create it? I left a comment on how to do that.

Reply:
Done